src/compass/schemas/s1_cslc_geo.yaml

```
@@ -91,6 +93,12 @@ range_split_spectrum_options:
   # Bandwidth of the high-band sub-image
   high_band_bandwidth: num(min=0, required=False)
 
+lut_options:
+  # LUT spacing in x/slant range direction
+  x_spacing: int(min=1, required=False)
+  # LUT spacing in y/azimuth direction
+  y_spacing: int(min=1, required=False)
```

```suggestion
  # LUT spacing in x/slant range direction in meters
  x_spacing: number(min=1, required=False)
  # LUT spacing in y/azimuth direction in seconds
  y_spacing: number(min=1, required=False)
```
In line with @seongsujeong's recommendation to use scientific notation. Also add units to documenting comments.

 Also applicable to radar schema.

---

Just making sure before applying this change. At the moment, the only corrections we have in the `s1-reader` can be computed by setting the spacing parameters in terms of number of pixels (see https://github.com/opera-adt/s1-reader/blob/71e2c9e589f9e010b4952a574fcda83806561339/src/s1reader/s1_burst_slc.py#L401). For the schema/runconfig, do we want to use meters and second? I guess inside `lut.py`, we will need to fetch the number of pixels by dividing the spacing allocated in the runconfig by the spacing of the burst in slant range and azimuth directions. We can get the range_pixel_spacing from the Sentinel1BurstSlc object but not the azimuth spacing (which either should be hard-coded or extracted at center scene?). What do you recommend @LiangJYu @seongsujeong ? I would prefer to keep it in terms of number of pixels to avoid unnecessary conversions (especially getting the spacing in the azimuth direction is not trivial)

---

I think we can use whichever makes more sense to calculate the correction, as long as we clearly mention about what that is. In case of FM mismatch rate, I stick with what ETAD ATBD is using for correction grid. (i.e. long time - short time).

---

That is interesting. So ETAD has seconds both in range and azimuth directions? I guess, and @LiangJYu can correct me if I am wrong, for the GSLC core module, we have used seconds for the LUT in azimuth and meters for the one along slant range?

---

> So ETAD has seconds both in range and azimuth directions?

Yes.

>I guess, and @LiangJYu can correct me if I am wrong, for the GSLC core module, we have used seconds for the LUT in azimuth and meters for the one along slant range?


That's my understanding about ISCE3. We use meters when defining range grid. See [here](https://github.com/opera-adt/s1-reader/blob/4000bc7a31d79e740212ab31495ca28ad0e2d426/src/s1reader/s1_burst_slc.py#L201-L209) and [here](https://github.com/opera-adt/s1-reader/blob/4000bc7a31d79e740212ab31495ca28ad0e2d426/src/s1reader/s1_reader.py#L569)
It would be great if @LiangJYu can confirm this.

---

> > So ETAD has seconds both in range and azimuth directions?
> 
> Yes.
> 
> > I guess, and @LiangJYu can correct me if I am wrong, for the GSLC core module, we have used seconds for the LUT in azimuth and meters for the one along slant range?
> 
> That's my understanding about ISCE3. We use meters when defining range grid. See [here](https://github.com/opera-adt/s1-reader/blob/4000bc7a31d79e740212ab31495ca28ad0e2d426/src/s1reader/s1_burst_slc.py#L201-L209) and [here](https://github.com/opera-adt/s1-reader/blob/4000bc7a31d79e740212ab31495ca28ad0e2d426/src/s1reader/s1_reader.py#L569) It would be great if @LiangJYu can confirm this.

Confirming ISCE3 core geocodeSlc has az in seconds and slant range in meters.